Vision/README.md

```diff
@@ -1,51 +1,47 @@
 # The W3C Vision: An Emphasis on Values
 
 ## Introduction
-The World Wide Web was conceived more than 25 years ago 
+The World Wide Web was originally conceived 
 as a tool for sharing information. 
-It has become much more than that; 
-it is a fundamental part of the lives of much of humanity, 
-enabling access to information, education, 
+It has evolved into a fundamental part of humanity, 
+providing access to knowledge, education, 
 commerce and shopping, social experiences, 
-civic functions, entertainment, and more. 
+civic duties, entertainment, and more.
 
-The Web is a force for good; 
-indeed, it has catalyzed major social changes. 
-At the same time, the Web's phenomenal success 
+The Web has been a force for good, 
+and has sparked major social changes. 
+But the Web's amazing success 
 has led to many unintended consequences 
```

Comment:
```suggestion
has also produced unintended consequences 
```

Reply:
Again, I'm trying to simplify the language whereever possible.

Reply:
OK. ("to-may-to, to-mah-to" - for me, "produce" is a simple direct verb and "lead to" is an indirect phrasing.)